code/__HELPERS/unsorted.dm

```diff
@@ -709,7 +710,7 @@ proc/GaussRandRound(var/sigma,var/roundto)
 					if(progbar)
 						progbar.loc = null
 			return 0
-		if ( user.loc != user_loc || target.loc != target_loc || user.get_active_hand() != holding || user.isStunned())
+		if ( user.loc != user_loc || target.loc != target_loc || (!user.is_holding_item(holding) && needs_item) || user.isStunned())
```

Comment:
put the needs_item before !user.is_holding_item

Reply:
why

Reply:
makes more sense, if(needs_item && has_item) rather than (has_item && needs_item)

Also a ~~micro optimization, but we don't talk about those when cluster is near~~